### ISSUE TITLE
Warning when precompiling Genie

### ISSUE THREAD

**Issue by shashikdm** [GenieFramework/Genie.jl#209]
**Describe the bug**
When I run using Genie, during precompilation, I get a warning as shown below.

**Error stacktrace**
```
julia> using Genie
[ Info: Precompiling Genie [c43c736e-a2d1-11e8-161f-af95117fbd1e]
┌ Warning: `@add_arg_table` is deprecated, use `@add_arg_table!` instead
└ @ ArgParse none:0
```

**To reproduce**
1. install Julia 1.0.5
2. ] add Genie
3. using Genie

**Expected behavior**
Precompile without warnings and errors

**Additional context**
```
julia> versioninfo()
Julia Version 1.0.5
Commit 3af96bcefc (2019-09-09 19:06 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, skylake)
```
```
(MyGenieApp) pkg> st
    Status `~/FluxDesigner/MyGenieApp/Project.toml`
  (empty environment)
```

**Comment by essenciary** [comment]
@shashikdm Thanks! One of the packages has been updated and now throws a warning. I'll look into it.

**Comment by essenciary** [comment]
Should be fixed in the new v0.26.1
